src/components/pages/Dashboard.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import PageTitle from '../atoms/PageTitle';
+import Tab from '../molecules/Tab';
+import TabsContainer from '../molecules/TabsContainer';
+
+interface IProps {
+  className?: string;
+}
+
+const UnstyledDashboard: React.FC<IProps> = props => {
+  return (
+    <div className={props.className}>
+      <div className="page-title">
+        <PageTitle
+          title="Dashboard"
+          description="Showing key performance indicators for current company"
+        />
+      </div>
+      <TabsContainer className="tabs-container">
+        <Tab label="Minimum Liquidity">
+          <h3>Projected minimum liquidity</h3>
+        </Tab>
+        <Tab label="Transaction Volumes">
+          <h3>Transaction volumes</h3>
+        </Tab>
+        <Tab label="Balances">
+          <h3>Monthly balances</h3>
+        </Tab>
+      </TabsContainer>
+    </div>
+  );
+};
+
+const Dashboard = styled(UnstyledDashboard)`
```

Comment:
You could just rename `UnstyledDashboard` to `Dashboard`, and export it directly instead.

```typescript
export default styled(Dashboard)`
  ...
`;
```

Reply:
Much nicer, thanks :+1: